tests/sentry/tasks/test_sentry_apps.py

```diff
@@ -879,6 +879,7 @@ def test_slow_should_disable(self, safe_urlopen):
     @patch(
         "sentry.utils.sentry_apps.webhooks.safe_urlopen", return_value=MockFailureResponseInstance
     )
+    @freezegun.freeze_time("2022-01-01 03:30:00")
```

Comment:
You should just be able to do `@freeze_time` without importing `freezegun` again